client/a8c-for-agencies/sections/team/hooks/use-member-list.ts

```diff
@@ -21,25 +20,18 @@ export function useMemberList() {
 		refetchMemberInvites();
 	}, [ refetchActiveMembers, refetchMemberInvites ] );
 
-	const members: TeamMember[] = useMemo( () => {
-		const data = [
-			...( activeMembers ?? [] ),
-			...( memberInvites?.map( ( invite ) => ( {
+	return {
+		refetch,
+		isPending: isActiveMembersPending || isMemberInvitesPending,
+		activeMembers: activeMembers ?? [],
+		invitedMembers:
+			memberInvites?.map( ( invite ) => ( {
 				id: invite.id,
 				displayName: invite.displayName,
 				email: invite.email,
 				avatar: invite.avatar,
 				status: invite.status as 'active' | 'pending' | 'expired',
-			} ) ) ?? [] ),
-		];
-
-		return data;
-	}, [ activeMembers, memberInvites ] );
-
-	return {
-		refetch,
-		isPending: isActiveMembersPending || isMemberInvitesPending,
-		members,
-		hasMembers: members.length > 1, // We exclude the owner from the count
+			} ) ) ?? [],
+		hasMembers: ( activeMembers && activeMembers.length > 1 ) || memberInvites?.length, // We exclude the owner from the count
```

Comment:
[NITPICK]

```suggestion
		hasMembers: !! (activeMembers?.length  || memberInvites?.length), // We exclude the owner from the count
```

Reply:
We will need to make sure active members count is greater than 1.

Reply:
I see. Sorry to miss that. We could do `activeMembers?. length > 1`. My suggestion was to use optional chaining here.

Reply:
I had trouble with optional chaining while comparing it with > 1. I end up with this since it is simpler than (activeMembers?.length ?? 0 > 1)